client/odie/message/index.tsx

```diff
@@ -11,10 +10,11 @@ import MaximizeIcon from 'calypso/assets/images/odie/maximize-icon.svg';
 import MinimizeIcon from 'calypso/assets/images/odie/minimize-icon.svg';
 import WapuuAvatar from 'calypso/assets/images/odie/wapuu-squared-avatar.svg';
 import WapuuThinking from 'calypso/assets/images/odie/wapuu-thinking.svg';
-import AsyncLoad from 'calypso/components/async-load';
 import Gravatar from 'calypso/components/gravatar';
 import { getCurrentUser } from 'calypso/state/current-user/selectors';
```

Comment:
We need to inject this as a component to represent the user's profile. It ca be a new prop for Odie, userAvatarComponent={<Gravatar... />}